src/test/regress/sql/alter_distributed_table.sql

```diff
@@ -280,5 +280,19 @@ CREATE MATERIALIZED VIEW mat_view AS SELECT * FROM mat_view_test;
 SELECT alter_distributed_table('mat_view_test', shard_count := 5, cascade_to_colocated := false);
 SELECT * FROM mat_view ORDER BY a;
 
+CREATE TABLE distributed_table_1 (
```

Comment:
Can we have a test `citus_local` is converted via `create_reference_table` and `create_distributed_table`?

Reply:
this depends on #4613 since we cannot have `generated as always stored` expressions on citus local table columns currrently :/

Reply:
since we are about to merge #4613, I will add that test before merging this pr

Reply:
fa3123c75d1dccd7c2762fbd6fd2de0f28a0c793